src/gatsby-theme-iterative-docs/sidebar.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const normalizeSidebar = require('../../plugins/gatsby-theme-iterative-docs/normalize-sidebar')
+const sidebar = require('../../content/docs/sidebar.json')
+module.exports = normalizeSidebar(sidebar)
```

Comment:
This shadowed file definition must be present and default export a normalized sidebar. It also must be ES5 format to be used in `gatsby-node` contexts like remark plugins and server redirects- while those can't do shadows, they can still require this file directly.